docs/book/src/user/quick-start.md

```diff
@@ -1456,6 +1456,39 @@ helm install --kubeconfig=./capi-quickstart.kubeconfig --repo https://raw.github
 
 For more information, see the [CAPZ book](https://capz.sigs.k8s.io/topics/addons.html).
 
+{{#/tab }}
+{{#tab OpenStack}}
+
+Before deploying the OpenStack external cloud provider, configure the `cloud.conf` file for integration with your OpenStack environment:
+
+```bash
+cat > cloud.conf <<EOF
+[Global]
+auth-url=<your_auth_url>
+application-credential-id=<your_credential_id>
+application-credential-secret=<your_credential_secret>
+region=<your_region>
+domain-name=<your_domain_name>
+EOF
```

Comment:
Could you link to the docs explaining the configuration too? They're here: https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md#config-openstack-cloud-controller-manager

Reply:
Yes thanks, it is done now.